js_modules/dagster-ui/packages/ui-components/src/components/Popover.tsx

```diff
@@ -25,7 +25,7 @@ export const GlobalPopoverStyle = createGlobalStyle`
         width: 16px;
         height: 16px;
         background: ${Colors.Gray900};
-        mask-image: url(${searchSVG});
+        mask-image: url(${searchSVG.src});
```

Comment:
Iiiinteresting, I think that  `DagsterCodeMirrorStyle` does this too and may also be broken 👀